functions/fzf-emoji.fish

```diff
@@ -29,17 +29,23 @@ function fzf-emoji
 
   # Select Emoji using fzf
   # copy command priority
+  #   - wl-copy
   #   - xsel
   #   - pbcopy
   #   - win32yank
-  if type "xsel" >/dev/null 2>&1
-      set --function copy "xsel --clipboard --input"
+  
+  set wl true
+  
+  if type "wl-copy" >/dev/null 2>&1
+      set --function copy "wl-copy"
```

Comment:
I think that `copy` should be `wl-copy -n`.

It looks like the `if` on L58 can be eliminated by making this change.